repos/tap/components/sidebar/sidebarContent.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import { useStoreItems } from 'SVHooks/store/useStoreItems'
+import { Values } from 'SVConstants'
+import {  
+  TreeList,
+  Text,
+  View,
+  TouchableIcon
+} from 'SVComponents'
+import { NoteAdd } from 'SVAssets/icons'
+import { useTheme } from '@keg-hub/re-theme'
+
+const { CATEGORIES, SIDEBAR_TYPES } = Values
+
+/**
+ * Title with Touchable NoteAdd icon
+ * 
+ * @param {object} props
+ * @param {object} props.styles
+ * @param {string} props.title
+ */
+const FileTreeHeader = ({styles, title}) => {
+  return (
+    <View style={styles?.main}>
+      <View style={styles?.textContainer}>
+        <Text 
+          style={styles?.text}
+        >
+          {title}
+        </Text>
+      </View>
+      <View style={styles?.iconContainer}>
+      <TouchableIcon
+        Component={(
+          <NoteAdd
+            size={styles?.icon?.size}
+            fill={styles?.icon?.fill}
+            stroke={styles?.icon?.stroke}
+          />
+        )}
+        onPress={() => console.log('---TODO: creating new file action----')}
+      />
+        
+      </View>
+    </View>
+  )
+}
+
+/**
+ * Manages the content displayed in the sidebar
+ */
+export const SidebarContent = () => {
```

Comment:
This is great. I really like how you've done it. 
We should extract the `FileTreeHeader` component to its own file.
* I imagine we will add more items to the `switch` statement, so we should keep it clean of other components
* Also move the `View` Wrapper and `TreeList` into the extracted component
* So the `switch` statement looks like this =>
```js
  switch (activeId) {
    case SIDEBAR_TYPES.FILE_TREE:
      return (
       <FileTreePanel {...props}  />
      )
```

* Then in a `fileTreePanel.js` file add =>
```js
export const FileTreePanel = props => {
  return (
        <View>
          <FileTreeHeader 
            styles={styles?.testFiles}
            title={'TEST FILES'}
          />
          <TreeList />
        </View>
  )
}
```

The Code is untested, but I think you get the idea